suite-native/module-onboarding/src/screens/AnalyticsConsentScreen.tsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import { useDispatch } from 'react-redux';
+
+import { useNavigation } from '@react-navigation/native';
+
+import {
+    OnboardingStackParamList,
+    OnboardingStackRoutes,
+    StackNavigationProps,
+} from '@suite-native/navigation';
+import { Button, Card, Stack, Text } from '@suite-native/atoms';
+import { Link } from '@suite-native/link';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { enableAnalyticsThunk } from '@suite-native/analytics';
+
+import { OnboardingScreen } from '../components/OnboardingScreen';
+import { AnalyticsInfoRow } from '../components/AnalyticsInfoRow';
+
+const buttonsWrapperStyle = prepareNativeStyle(() => ({
+    width: '95%',
```

Comment:
`Button`s on other screens does not define such a `width`. Please unify it across the screens

Reply:
[411b2ef](https://github.com/trezor/trezor-suite/pull/7889/commits/411b2ef4d48df39092779f8f118b3b7d57b00cc0)